xenit-aps/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: alfresco
+name: activiti
 description: A Helm chart for Alfresco managed by Xenit Solutions
```

Comment:
The description should mention that this helm chart is for APS. Now the description is the same as the one from helm-alfresco.